Source/WebKit/_WebKit_SwiftUI/API/WebView.swift

```diff
@@ -67,19 +67,29 @@ public struct WebView: View {
 
     private let storage: Storage
 
-    // swift-format-ignore: AllPublicDeclarationsHaveDocumentation
-    public var body: some View {
+    @ViewBuilder
```

Comment:
For my own understanding, do we need `@ViewBuilder` here because it returns a different value depending on the platform?

Otherwise, I thought it was only needed when returning a tuple of Views.